ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/KnownPeers.hs

```diff
@@ -172,16 +187,18 @@ jobPeerShare PeerSelectionActions{requestPeerShare}
         Right totalResults ->
           return $ Completion $ \st _ ->
            let peerResults = zip peers totalResults
-               -- Filter known-to-be ledger peers before adding them to known
+               -- Filter known-to-be big ledger peers before adding them to known
                -- peers. We to improve efficacy of Peer Sharing, because ledger
                -- peers are going to be hub nodes in the network (i.e. nodes
                -- with higher valency) they are going to be shared quite often,
                -- which means other smaller valency nodes won't be shared that
                -- much. So any chance we can have of filtering them from peer
                -- sharing results is welcome.
```

Comment:
We should change `ledger peers -> big ledger peers` further in the comment too.  I'd also add the reason why we are not excluding ledger peers.  This will help smaller SPOs, so they can be discovered by the ledger and peer sharing.

Note that in the future we might try to make the ledger peers be selected uniformly (it was suggested by cryptographers), then we can also exclude ledger peers from peer sharing.

Reply:
> Note that in the future we might try to make the ledger peers be selected uniformly (it was suggested by cryptographers), then we can also exclude ledger peers from peer sharing.

Lets burn that bridge when we get to it. As we have discussed previously there are lots of incorrectly setup or offline peers belonging to pools with very low stake.

Reply:
I've moved all comments about the filtering of peers to the comment for `jobPeerShare` function.